src/hook/pauseResume.js

```diff
@@ -0,0 +1,318 @@
+/**
+* Copyright (c) 2015 NAVER Corp.
+* egjs projects are licensed under the MIT license
+*/
+eg.module("pauseResume", ["jQuery"], function($) {
+	"use strict";
+
+	var animateFn = $.fn.animate;
+	var stopFn = $.fn.stop;
+	var uuid = 1;
+	var getStyles;
+
+	if (window.getComputedStyle) {
+		getStyles = function(elem) {
+			var view = elem.ownerDocument.defaultView;
+
+			if (!view || !view.opener) {
+				view = window;
+			}
+
+			return view.getComputedStyle(elem);
+		};
+	} else if (document.documentElement.currentStyle) {
+		getStyles = function(elem) {
+			return elem.currentStyle;
+		};
+	}
+
+	function AniProperty(el, prop, optall, prevProp) {
+		this.opt = optall;
+		this.start = -1;
+		this.elapsed = 0;
+		this.paused = false;
+		this.uuid = uuid++;
+		this.easingNames = [];
+		this.prop = $.extend({}, prevProp);
+		var computed;//It's only used when relative value is applied.
+
+		for (var propName in prop) {
+			var propValue = prop[propName];
+			var markIndex;
+			var sign;
+
+			//If it has a absoulte value.
+			if (typeof propValue !== "string" ||
+				(markIndex = propValue.search(/[+|-]=/)) < 0) {
+				this.prop[propName] = propValue;
+				continue;
+			}
+
+			//If it has a relative value
+			sign = propValue.charAt(markIndex) === "-" ? -1 : 1;
+
+			if (!prevProp[propName]) {
+				// If this makes performance issues, then change the way of calucating relative value.
+				computed = computed || getStyles(el);
```

Comment:
If ```prop``` has a more than one property, then ```computed``` can be used.

For example, prop has below.
```
prop = {
  left: "+=100px",
  width: "+=100px",
  height: "+=100px"
}
```
Without computed, below code can call getStyles 3 times in above case.

```
var computed;
for (var propName in prop) {
  //...
  if (!prevProp[propName]) {
    computed = computed || getStyles(el);
    //...
  }
  //...
}
```